code/modules/jobs/job_types/clown.dm

```diff
@@ -59,3 +59,4 @@
 	H.dna.add_mutation(CLOWNMUT)
 	for(var/datum/mutation/human/clumsy/M in H.dna.mutations)
 		M.mutadone_proof = TRUE
+	H.mind.adjust_experience(/datum/skill/pie_throwing, 1000) // clown college
```

Comment:
aw, so mimes won't start with pie-ing experience?

Reply:
mimes are clowns that didnt go to school

Reply:
Just because you HATE MIMES!!!one!!!1 doesn't mean that they shouldn't be allowed to join in on the cream pie-ing fun, IMO.